lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -903,7 +906,9 @@ int sgw_handle_modify_bearer_request(
 
         // Send end marker to eNB and then delete the tunnel if enb_ip is
         // different
-        if (is_enb_ip_address_same(
+        if (does_bearer_context_hold_valid_enb_ip(
```

Comment:
In which case eps_bearer_ctxt does not hold IP address ?
If the IP address is zero, how come SPGW created tunnel to this eNB?

Reply:
As part of create session request, spgw creates bearer context but doesn't have eNB's S1-U IP address, this is received in modify bearer request. So on arrival of modify bearer request, eps_bearer_context doesn't hold eNB's S1-U IP address. While handling modify bearer request, eNB's S1-U IP address gets updated to eps_bearer_context.
Apart from initial attach procedure, modify bearer procedure is initiated as part of handover procedure, in which case eps_bearer_context holds valid eNB's S1-U IP address. During this procedure, spgw shall delete the old rule and adds new rule to OVS